Gibe.DittoProcessors/Processors/GridAttribute.cs

```diff
@@ -44,7 +45,7 @@ public override object ProcessValue()
 						break;
 					case GridEditorAliases.Media:
 						var gridContentMediaValue = JsonConvert.DeserializeObject<GridContentMediaValue>(control.Value.ToString());
-						var mediaImage = _mediaService.GetImage(gridContentMediaValue.Id);
+						var mediaImage = _mediaService.GetImage<CaptionMediaImageModel>(gridContentMediaValue.Id);
```

Comment:
Is this the default for Grid that it'll be a CaptionMediaImageModel or is that Bristol Sport specific